cmake/SetupTPLs.cmake

```diff
@@ -19,6 +19,26 @@ IF(NOT DEAL_II_WITH_TRILINOS)
           "Error! deal.II must be compiled with Trilinos support.")
 ENDIF()
 
+INCLUDE(CheckCXXSourceCompiles)
+SET(CMAKE_REQUIRED_INCLUDES ${DEAL_II_INCLUDE_DIRS})
+CHECK_CXX_SOURCE_COMPILES(
```

Comment:
Trilinos include directories come from the deal.II variable?

Reply:
Yes, we find `Trilinos` only through `deal.II`.